src/site/layouts/home.drupal.liquid

```diff
@@ -14,8 +14,18 @@
         <div class="hub-links-container" data-e2e="bucket">
           <h2 class="heading-level-3 hub-links-title"><i class="icon-large-baseline icon-heading hub-icon-{{ card.label | downcase | replace: ' ', '-' }} hub-color-{{ card.label | downcase | replace: ' ', '-' }}"></i>{{ card.label }}</h2>
           <ul class="hub-links-list">
+            {% assign cardLabel = card.label %}
             {% for link in card.links %}
-              <li><a data-nav-path="{{ card.label }}->{% if link.url.path != empty %}{{ link.url.path }}{% else %}{{ link.label }}{% endif %}" href="{{ link.url.path }}">{{ link.label }}</a></li>
+              <li>
+                <a href="{{ link.url.path }}"
+                  onclick="recordEvent({
+                    event: 'nav-zone-one',
+                    'nav-path': '{{ cardLabel }}->{% if link.url.path != empty %}{{ link.url.path }}{% else %}{{ link.label }}{% endif %}',
```

Comment:
This is subjective, but do you think `{{ card.label }}` to avoid the assignment line above is easier to read?

Reply:
I do, but the Liquid scoping couldn't find `card.label` so it produced the empty string 🤷‍♂

Reply:
Ugggh liquid... 🤦‍♂